### DISCUSSION TITLE
Access to iOS Library directory

### DISCUSSION THREAD

**Discussion by pip108** [ionic-team/capacitor#3073] [Ideas]
In the filesystem plugin for iOS, getDirectory returns (by defaulting) `.documentDirectory` for "DATA". Returning `.libraryDirectory` for "DATA" would be a better solution.

```diff
   func getDirectory(directory: String) -> FileManager.SearchPathDirectory {
     switch directory {
+    case: "DATA":
+      return .libraryDirectory
     case "DOCUMENTS":
       return .documentDirectory
     case "CACHE":
       return .cachesDirectory
     default:
       return .documentDirectory
     }
   }
```

**Comment by imhoffd** [comment]
I think this would be a breaking change. Apps that are expecting to find files in the documents directory using "DATA" will suddenly not be able to find them if they update Capacitor.

**Comment by robingenz** [comment]
I have created a PR: https://github.com/ionic-team/capacitor-plugins/pull/666
